libs/ledger/tests/notarisation/notarisation_tests.cpp

```diff
@@ -82,7 +82,7 @@ struct NotarisationNode
     , reactor{"ReactorName" + std::to_string(index)}
     , muddle_certificate{CreateNewCertificate()}
     , muddle{muddle::CreateMuddleFake("Test", muddle_certificate, network_manager, "127.0.0.1")}
-    , chain{false, ledger::MainChain::Mode::IN_MEMORY_DB}
+    , chain{false, ledger::MainChain::Mode::IN_MEMORY_DB, true}
```

Comment:
Seems like we should go for the stutter thing since the tests all pass? Can we review IRL the contents of the main chain tests since it is a lot of code?